contrib/runners/inquirer/runner.yaml

```diff
@@ -0,0 +1,37 @@
+- description: A runner for Inquiries
+  enabled: true
+  name: inquirer
+  runner_module: inquirer
+  runner_parameters:
+    schema:
+      default:
+        title: response_data
+        type: object
+        properties:
+          continue:
+           type: boolean
+           description: Would you like to continue the workflow?
+        required: ["continue"]
```

Comment:
We actually use a different / updated json schema where "required" attribute is part of the property (originally it's a global attribute like you have done it, here but we made changes to the schema a long time ago so we can use per attribute / property definition).

In this case this means it would be part of "continue" property:

```python
continue:
  type: boolean
   description: Would you like to continue the workflow?
   required: true
```

Reply:
Oh boy, the old syntax is confusing as hell.

Reply:
Fixed in https://github.com/StackStorm/st2/commit/aa7b1d479a439c8502b1a8775534f157b6a35917

Reply:
Actually I misunderstood this and realized later that I needed to use the built in schema validation code. I did this (and corrected any straggling v4-style required fields) in https://github.com/StackStorm/st2/pull/3653/commits/e618dee8ccb7496431fabbd3b54fc9c60db02029 - hopefully that's a bit better.